operation_test.go

```diff
@@ -244,6 +244,33 @@ func TestOperation_ParseResponseWithDefault(t *testing.T) {
 	assert.Equal(t, "A response", operation.Responses.StatusCodeResponses[200].Description)
 }
 
+func TestParseResponsePrimitiveTypeWithFormat(t *testing.T) {
+	t.Parallel()
+
+	comment := `@Success 200 {integer} int16 "response with format"`
+	operation := NewOperation(nil)
+	err := operation.ParseComment(comment, nil)
+	assert.Error(t, err)
+
+	comment = `@Success 200 {integer} int32 "response with format"`
```

Comment:
This is a stupid corner case , Can you show me an example how you send and receive an int32 over an HTTP request ? :) 
Usually the response is a JSON object, not a string value :flashlight: